src/core/terms.ml

```diff
@@ -263,6 +263,34 @@ type ctxt = (term Bindlib.var * term * term option) list
 (** Type of a list of unification constraints. *)
 type unif_constrs = (ctxt * term * term) list
 
+(** Some definitions for unification hints. *)
```

Comment:
More explanations would be welcome here. Mainly, you define symbols that will be used to rewrite unification constraints. Why not moving this module in unif.ml?